src/FluentResults/Logging/IResultLogger.cs

```diff
@@ -3,7 +3,7 @@ namespace FluentResults
 {
     public interface IResultLogger
     {
-        void Log(string context, ResultBase result);
-        void Log<TContext>(ResultBase result);
+        void Log(string context, ResultBase<IError> result);
+        void Log<TContext>(ResultBase<IError> result);
```

Comment:
There are braking changes.
But I've read somewhere that you allowe some braking changes in v3.